docs/notes/2.27.x.md

```diff
@@ -43,6 +43,8 @@ Added support for formatting typescript and tsx files with Prettier.
 
 Fixed a bug where `pnpm-workspaces.yaml` could affect NPM or Yarn projects - it should be ignored.
 
+Allow extra env vars during nodejs package manager installation through `[nodejs].package_manager_extra_env_vars` option or and the `extra_env_vars` field on `package_json` targets.
```

Comment:
Minor copy edits:

- adding links to the field docs
- adjusting "or and" -> "and/or"
```suggestion
Allow extra env vars during nodejs package manager installation through [the `[nodejs].package_manager_extra_env_vars` option](https://www.pantsbuild.org/2.27/reference/subsystems/nodejs#package_manager_extra_env_vars) and/or [the `extra_env_vars` field](https://www.pantsbuild.org/2.27/reference/targets/package_json#package_manager_extra_env_vars) on `package_json` targets.
```